ocaml/xapi/xapi_xenops.ml

```diff
@@ -435,7 +435,7 @@ module MD = struct
             let rate = Int64.of_string (List.assoc "kbps" qos_params) in
             Some (rate, timeslice)
           with
-          | Failure "int_of_string" ->
+          | Failure _ (* int_of_string *) ->
```

Comment:
Just for reference. If we chang these functions in the future we should take into account eventual other failures.